hack-day.md

```diff
@@ -41,6 +41,8 @@ cd core-command
 gh repo fork
 ```
 
+We have started a video tutorial series that you can watch here: [WP-CLI Contribution Tutorials](https://www.youtube.com/playlist?list=PL_B8Y6K6MH2d6T7pYa6dloUgk67mfBT4K). These videos will provide an overview of the packages and then go through setting up the development environment so allow you to run automated tests locally.
```

Comment:
May be a small typo. `environment so allow` -> `environment to allow`

Reply:
Thanks, resolved.